src/Umbraco.Cms.Targets/Umbraco.Cms.Targets.csproj

```diff
@@ -58,4 +60,9 @@
       <_PackageFiles Include="$(IntermediateOutputPath)_._" PackagePath="lib\$(TargetFramework)" />
     </ItemGroup>
   </Target>
+
```

Comment:
I think we lack a clean on remove like we have for appsettings

Reply:
Yes you are right.